.gitignore

```diff
@@ -7,6 +7,7 @@ node_modules/
 coverage/
 .idea/
 .project/
+.nx
```

Comment:
Is this something NX added in v17?

Reply:
That's right. https://github.com/nrwl/nx/releases/tag/17.0.0
> core: update default cache directory to .nx/cache (https://github.com/nrwl/nx/pull/19536)

I tried using the [`cacheDirectory`](https://nx.dev/recipes/running-tasks/change-cache-location) option, but the `.nx/cache` directory still gets created.